guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -10,17 +10,18 @@ The following table details storage requirements for specific directories.
 These values are based on expected use case scenarios and can vary according to individual environments.
 
 ifdef::katello,satellite[]
-The runtime size was measured with {RHEL} 6, 7, and 8 repositories synchronized.
+The runtime size was measured with {RHEL} 6, 7, and 8 Client repositories synchronized.
```

Comment:
This is an incorrect change. It was tested with real RHEL synced, which is much larger.
```suggestion
The runtime size was measured with {RHEL} 6, 7, and 8 repositories synchronized.
```

Reply:
Addressed :+1: